docs/Contribute.md

```diff
@@ -0,0 +1,32 @@
+# Contribution Guidelines
+
+Thanks for taking the time to contribute!
+
+The following is a set of guidelines for contributing to SecDevLabs. These are just guidelines, not rules, so use your best judgement and feel free to propose changes to this document in a pull request! 😃
+
+## So... I built this app, it looks great, but how can I send it to SecDevLabs?
+
+Well, a good place to start is to make sure you can deploy your app with Docker-Compose. All the apps in SecDevLabs are built with it in order to ensure you can get get right into the action! ✓
+
+So your app can be deployed with Docker-Compose, nice job! But wait, does it have a Makefile? If it doesn't, be sure to have a look [here][1], we have a template you can follow to create your own! ✓
+
+## Ok, my app is looking awesome and all set up with Docker-Compose, what else do I need to send it?
+
+Do you have an attack narrative? Well, if you don't, worry not, we got you! Have a look at this [attack narrative template][2] and you will be able to write your own in no time! ✓
```

Comment:
We could also create an APP Template page! What do you think? 😃

Reply:
What do you mean @rafaveira3 ? You are suggesting we could have a standard app so that people could just modify it to fit their own app?

Reply:
Yeah ! A template that similar to this [README](https://github.com/globocom/secDevLabs/tree/master/owasp-top10-2017-apps/a5/ecommerce-api).

Reply:
Sure @rafaveira3 !